File: python/darts/darts.py
"""exercise"""
import math

def score(x, y):
    """
    :param x:
    :param y:
    :return:
    """

    dist = math.sqrt(x**2 + y**2)

    if dist <= 1:
        result = 10
    elif dist <= 5:
        result = 5
    elif dist <= 10:
        result = 1
    else:
        result = 0

    return result

File: java/darts/src/main/java/Darts.java
class Darts {
    int score(double xOfDart,double  yOfDart) {
        var dist = Math.sqrt(Math.pow(xOfDart,2) + Math.pow(yOfDart,2));
        var s = 0;
        if( dist <= 1){
            s = 10;
        } else if (dist <= 5){
            s = 5;
        }
        else if ( dist <= 10){
            s = 1;
        }
        return s;
    }
}
